deploy/manifests/test-sts.yaml

```diff
@@ -41,7 +41,7 @@ spec:
   - metadata:
       name: volume
     spec:
-      storageClassName: microk8s-hostpath
+      storageClassName: hostpath-storage
```

Comment:
Not very important, but maybe `storageClassName: hostpath-storage` stutters, how about calling this `hostpath` instead?